package/kedro_viz/server.py

```diff
@@ -349,6 +349,7 @@ def _call_viz(
     if save_file:
         Path(save_file).write_text(json.dumps(data, indent=4, sort_keys=True))
     else:
-        if browser:
-            webbrowser.open_new("http://127.0.0.1:{:d}/".format(port))
+        is_localhost = host in ("127.0.0.1", "localhost", "0.0.0.0")
+        if browser and is_localhost:
```

Comment:
Could this change the user experience at all from the next release (when they pass --browser flag to non-local host)? If so, it's worth mentioning in the release note (but I can't think of the cases...).

Reply:
You can also update line 269 and say that it only works for local host

Reply:
Updated both